[2025-02-18T04:31:03.750+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-18T04:31:03.766+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: my_first_dag.hello_task scheduled__2024-03-22T00:00:00+00:00 [queued]>
[2025-02-18T04:31:03.772+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: my_first_dag.hello_task scheduled__2024-03-22T00:00:00+00:00 [queued]>
[2025-02-18T04:31:03.773+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-02-18T04:31:03.815+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): hello_task> on 2024-03-22 00:00:00+00:00
[2025-02-18T04:31:03.829+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=338) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-02-18T04:31:03.830+0000] {standard_task_runner.py:72} INFO - Started process 348 to run task
[2025-02-18T04:31:03.837+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'my_first_dag', 'hello_task', 'scheduled__2024-03-22T00:00:00+00:00', '--job-id', '36', '--raw', '--subdir', 'DAGS_FOLDER/my-first-dag.py', '--cfg-path', '/tmp/tmpoj9swnyl']
[2025-02-18T04:31:03.841+0000] {standard_task_runner.py:105} INFO - Job 36: Subtask hello_task
[2025-02-18T04:31:03.878+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:209 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-02-18T04:31:03.934+0000] {task_command.py:467} INFO - Running <TaskInstance: my_first_dag.hello_task scheduled__2024-03-22T00:00:00+00:00 [running]> on host 3420485c558b
[2025-02-18T04:31:03.936+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:470 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-02-18T04:31:04.029+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='my_first_dag' AIRFLOW_CTX_TASK_ID='hello_task' AIRFLOW_CTX_EXECUTION_DATE='2024-03-22T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-03-22T00:00:00+00:00'
[2025-02-18T04:31:04.030+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-18T04:31:04.040+0000] {logging_mixin.py:190} INFO - Hello, Airflow!
[2025-02-18T04:31:04.040+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-02-18T04:31:04.055+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-18T04:31:04.056+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=my_first_dag, task_id=hello_task, run_id=scheduled__2024-03-22T00:00:00+00:00, execution_date=20240322T000000, start_date=20250218T043103, end_date=20250218T043104
[2025-02-18T04:31:04.101+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-18T04:31:04.113+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-18T04:31:04.114+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2025-02-18T16:34:06.256+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-18T16:34:06.272+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: my_first_dag.hello_task scheduled__2024-03-22T00:00:00+00:00 [queued]>
[2025-02-18T16:34:06.289+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: my_first_dag.hello_task scheduled__2024-03-22T00:00:00+00:00 [queued]>
[2025-02-18T16:34:06.292+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-02-18T16:34:06.314+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): hello_task> on 2024-03-22 00:00:00+00:00
[2025-02-18T16:34:06.321+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=1623) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-02-18T16:34:06.323+0000] {standard_task_runner.py:72} INFO - Started process 1632 to run task
[2025-02-18T16:34:06.322+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'my_first_dag', 'hello_task', 'scheduled__2024-03-22T00:00:00+00:00', '--job-id', '404', '--raw', '--subdir', 'DAGS_FOLDER/my-first-dag.py', '--cfg-path', '/tmp/tmpyd0bwkzl']
[2025-02-18T16:34:06.324+0000] {standard_task_runner.py:105} INFO - Job 404: Subtask hello_task
[2025-02-18T16:34:06.335+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:209 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-02-18T16:34:06.382+0000] {task_command.py:467} INFO - Running <TaskInstance: my_first_dag.hello_task scheduled__2024-03-22T00:00:00+00:00 [running]> on host 8720d9c228eb
[2025-02-18T16:34:06.385+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:470 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-02-18T16:34:06.453+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='my_first_dag' AIRFLOW_CTX_TASK_ID='hello_task' AIRFLOW_CTX_EXECUTION_DATE='2024-03-22T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-03-22T00:00:00+00:00'
[2025-02-18T16:34:06.454+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-18T16:34:06.465+0000] {logging_mixin.py:190} INFO - Hello, Airflow!
[2025-02-18T16:34:06.466+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-02-18T16:34:06.477+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-18T16:34:06.478+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=my_first_dag, task_id=hello_task, run_id=scheduled__2024-03-22T00:00:00+00:00, execution_date=20240322T000000, start_date=20250218T163406, end_date=20250218T163406
[2025-02-18T16:34:06.511+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-18T16:34:06.524+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-18T16:34:06.525+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
